spec/services/proofing_session_async_result_spec.rb

```diff
@@ -21,14 +21,4 @@
       expect(loaded_result.result).to eq(idv_result.deep_symbolize_keys)
     end
   end
-
-  context 'proofing timed out' do
-    let(:idv_result) { { errors: {}, messages: ['some message'], timed_out: true } }
-
-    it 'indicates a timeout' do
-      result = ProofingSessionAsyncResult.new(id: id, status: status, result: idv_result)
-
-      expect(result.timed_out?).to be
-    end
```

Comment:
ditto, keep the test but maybe update the assertion?

Reply:
These tests were both added in https://github.com/18F/identity-idp/pull/5642 to test the `|| result[:timed_out]` behavior, so I don't know that keeping them in some form adds too much in terms of testing.

I did re-add https://github.com/18F/identity-idp/pull/6013/files#diff-5e1bd7260032bcb24c2c1e2c64303586cde97665d52eec35e8294108f1b332c4R28-R31 back, which should cover this path in the code

Reply:
sweet, thanks!